IFC4x3/Properties/r/RailGap_0$Ooq6bAz7GfDzJmMijf3H/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="RailGap_0Ooq6bAz7GfDzJmMijf3H" Name="RailGap" UniqueId="3f632d06-94af-4742-937d-4f05acb690d1" PrimaryDataType="IfcPositiveLengthMeasure" />
```

Comment:
`NominalLength` or even derive from the geometry?

Reply:
Not possible here. It is a realy specific property requirement, if using NominalLength, the sense wouldn't be extracted from the pset.